Exiled.API/Enums/Side.cs

```diff
@@ -49,7 +49,7 @@ public enum Side
         Tutorial,
 
         /// <summary>
-        /// No team. Same as <see cref="Team.Dead"/>.
+        /// No team. Same as <see cref="Team.Dead"/>, <see cref="RoleTypeId.Overwatch"/> and <see cref="RoleTypeId.Filmmaker"/>.
```

Comment:
These RoleTypeIds are in Dead team, so it's useless

Reply:
Removed